### ISSUE TITLE
got weird prepend comma

### ISSUE THREAD

**Issue by lifesign** [jsoncomma/sublime-jsoncomma#14]
got this weird prepend comma today.  jsoncomma version `v0.2.6`

Input: 
```
{
  "autoload": {
    "psr-4": {
      "Kvothe\\KeyboardShortcuts\\": "src/"
    }
  },
  "extra": {
    "flarum-extension": {
      "title": "Keyboard Shortcuts",
      "icon": {
        "name": "fas fa-keyboard",
        "backgroundColor": "#333",
        "color": "#eee"
      }
    },
    "flagrow": {
      "discuss": ""
    }
  }
}
```
when i save file, it became like this

```
{
  "autoload": {
    "psr-4": {
      "Kvothe\\KeyboardShortcuts\\": "src/"
    }
  },
  "extra": {
    ",flarum-extension": {
      ",title": "Keyboard Shortcuts",
      "icon": {
        ",name": ",fas fa-keyboard",
        "backgroundColor": "#333",
        "color": "#eee"
      }
    },
    ",flagrow": {
      "discuss": ","
    }
  }
}

```

**Comment by jakeparis** [comment]
wired = weird

**Comment by lifesign** [comment]
oops, fixed~

**Comment by math2001** [comment]
Oh my god. I actually realized last night before going to bed that I forgot about escaping (the part causing the problem is that you are escaping `\` before a `"`) So, it shifts everything and inserts commas in the wrong place. I don't think it should be too hard to fix though :slightly_smiling_face:

**Comment by math2001** [comment]
Sorry for the inconvenience... It's fixed in [`jsoncomma v0.1.9`](https://github.com/jsoncomma/jsoncomma/releases/tag/v0.1.9) (unless you installed jsoncomma manually, everything will be updated for you :slightly_smiling_face: )
